### PR TITLE
Version 0.0.3

### PR DESCRIPTION
**Pull request by vincentsarago**

### CODE REVIEW

File: 0.0.3/Readme.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    "tiles": {
+        "030130": [
+            "s3://my-bucket/dir/file1.tif",
+            "s3://my-bucket/dir/file2.tif",
+        ]
+    },
+
+    // OPTIONAL. Default: null.
+    // A string describing the type of the asset.
+    "asset_type": "COG",
```
Comment (vincentsarago) [inline 0.0.3/Readme.md:98]: TBD

File: 0.0.3/Readme.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+    // OPTIONAL. Default: null.
+    // Asset's datatype info.
+    "data_type": "uint8",
```
Comment (vincentsarago) [inline 0.0.3/Readme.md:102]: TBD - allow gdal/rasterio/numpy like datatype

File: 0.0.3/Readme.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+    // OPTIONAL. Default: null.
+    // List of indexes.
+    // examples: (band1, band2, band3); (1, 2, 3); (red, green, blue)
+    "indexes": [1, 2, 3, 4],
```
Comment (vincentsarago) [inline 0.0.3/Readme.md:107]: 🤷‍♂️

File: 0.0.3/Readme.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+    // OPTIONAL. Default: null.
+    "band_descriptions": [(1, ""), (2, ""), (3, ""), (4, "")],
```
Comment (vincentsarago) [inline 0.0.3/Readme.md:110]: TBD

File: 0.0.3/Readme.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+    // OPTIONAL. Default: null.
+    // GDAL like colormap in form of {key: [r, g, b, a], ...}
+    "colormap": {},
```
Comment (vincentsarago) [inline 0.0.3/Readme.md:114]: ref #15

**Comment by kylebarron** [comment]
If there's going to be a new revision of the mosaicjson spec, I'd suggest an optional `path_prefix` key, defined as a prefix to every value of `tiles`. I think that would be the single biggest change to make JSON file sizes much smaller, since usually every key is prefixed by the same text, e.g.

```
s3://opendata.remotepixel.ca/dg_post_idai/2019_03_20/
```

in your example json.

File: 0.0.3/Readme.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+    // OPTIONAL. Default: null.
+    // The first value is the longitude, the second is latitude (both in
+    // WGS:84 values), the third value is the zoom level as an integer.
+    // Longitude and latitude MUST be within the specified bounds.
+    // The zoom level MUST be between minzoom and maxzoom.
+    // Implementations can use this value to set the default location. If the
+    // value is null, implementations may use their own algorithm for
+    // determining a default location.
+    "center": [ -76.275329586789, 39.153492567373, 8 ]
```
Comment (fvanderbiest) [inline 0.0.3/Readme.md:83]: same here

File: 0.0.3/schema.json [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+        "center": {
+            "type": "array",
+            "items": {
+                "type": "number"
+            }
+        },
+        "tiles": {
+            "type": "object",
+            "patternProperties": {
+                "[0-9]+": {
```
Comment (philvarner) [inline 0.0.3/schema.json:52]: A tighter regex for quadkeys is `[0123]{1,30}`

File: 0.0.3/Readme.md (hunk 4 above)
Comment (vincentsarago) [reply 0.0.3/Readme.md]: I've removed this in the latest commit.

File: 0.0.3/Readme.md (hunk 3 above)
Comment (vincentsarago) [reply 0.0.3/Readme.md]: I removed this in the latest commit.

File: 0.0.3/Readme.md (hunk 2 above)
Comment (vincentsarago) [reply 0.0.3/Readme.md:119]: I added a list of supported types (stoled from https://github.com/stac-extensions/raster#data-types)

File: 0.0.3/Readme.md (hunk 1 above)
Comment (vincentsarago) [reply 0.0.3/Readme.md:98]: More: I see `asset_type` being either `COG` or `STAC`. This is optional and `informative`.

**Comment by vincentsarago** [comment]
🥳 This is ready for review (only took 3years 😅). 

Latest additions:
- `tilematrixset`: TileMatrixSet document used to get the quadkeys
- `layers`: define a set of `layer` configuration to create tiles (e.g `true_color`)

**Pull request merged**